draft-ietf-tls-md5-sha1-deprecate-07.xml

```diff
@@ -173,9 +173,10 @@
    <section anchor="Signature_algorithms" title="Signature Algorithms">
    <t>
    Clients MUST include the signature_algorithms extension. Clients MUST NOT include MD5
-   and SHA-1 in it this extension. If a client does not send a signature_algorithms
-   extension, then the server MUST abort the handshake and send a handshake_failure alert,
-   except when digital signatures are not used (for example, when using PSK ciphers).
+   and SHA-1 in it this extension. Servers MUST NOT include MD5 and SHA-1 in ServerKeyExchange 
+   messages. If no other signature algorithms are available (for example, if the client does 
+   not send a signature_algorithms extension), the server MUST abort the handshake with a 
+   handshake_failure alert or select a different cipher suite.
```

Comment:
Oh, I meant we could move it down to line 186 below, which has the side effect of implicitly conditioning on when you would send ServerKeyExchange. "Servers MUST NOT include..." is the first sentence of that section.

Reply:
Oh, duh. I forgot we had a section for that.